README.md

```diff
@@ -32,6 +32,7 @@ For other operating systems, please check the [installation guide](https://solan
 After installing the compiler, write the following to flipper.sol:
 
 ```solidity
+@program_id("F1ipperKF9EfD821ZbbYjS319LXYiBmjhzkkf5a26rC") // For Substrate, this line must be removed.
```

Comment:
I don't like this either. What about providing two different versions of the file for both targets (e.g. move it to `Build for Solana` and `Build for Substrate` accordingly)?